spec/message.abnf

```diff
@@ -23,13 +23,18 @@ expression = literal-expression / variable-expression / annotation-expression
 literal-expression = "{" [s] literal [s annotation] [s] "}"
 variable-expression = "{" [s] variable [s annotation] [s] "}"
 annotation-expression = "{" [s] annotation [s] "}"
-annotation = (function *(s option))
+annotation = function
            / private-use-annotation
            / reserved-annotation
 
+markup       = "{" [s] markup-open [s] ["/"] "}"
+             / "{" [s] markup-close [s] "}"
+markup-open  = "#" identifier *(s option)
+markup-close = "/" identifier
+
 literal = quoted / unquoted
 variable = "$" name
-function = (":" / "+" / "-") identifier
+function = ":" identifier *(s option)
```

Comment:
Please move `*(s option)` back to the `annotation` production. It's helpful that `variable` and `function` define *names*. In fact, we should consider renaming them to `veriable-name` and `function-name`, respectively.

Reply:
On the other hand, isn't it also useful for a _function_ to have _options_? Previously, they were only a part of the _annotation_, which could effectively have any shape as _private-use_ and _reserved annotations_.